alchemy_news/README.md

```diff
@@ -0,0 +1,152 @@
+# Integrating the Watson Developer Cloud AlchemyAPI News Service with Watson Explorer
```

Comment:
There are many references to "Watson Developer Cloud", WDC, and Bluemix.  Most can be dropped if we are pointing right at the Alchemy service now.